extensions/lisp-mode/macroexpand.lisp

```diff
@@ -146,7 +146,7 @@
     (backward-up-list point)
     (skip-chars-backward point #'syntax-expr-prefix-char-p))
   (values point
-          (form-offset (copy-point point :temporary) 1)))
+           (form-offset (copy-point point :temporary) 1)))
```

Comment:
Is the indentation out of alignment?

Reply:
This seems to be a bug in `#'indent-buffer`, It has been happening in other spots too.

[Screencast from 2024-02-05 18-28-40.webm](https://github.com/lem-project/lem/assets/43155857/a0399e2b-475e-4ae5-bddd-e335ab9e4e61)